addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -151,9 +151,17 @@
             <DatiOrdineAcquisto t-if="record.ref and not record.reversed_entry_id">
                 <IdDocumento t-esc="format_alphanumeric(record.ref[:20])"/>
             </DatiOrdineAcquisto>
-            <DatiFattureCollegate t-if="record.reversed_entry_id">
-                <IdDocumento t-out="format_alphanumeric(record.reversed_entry_id.name[-20:])"/>
-                <Data t-out="format_date(record.reversed_entry_id.invoice_date)"/>
+            <DatiFattureCollegate
+                t-foreach="record.line_ids.mapped('matched_credit_ids.credit_move_id.move_id')"
```

Comment:
```suggestion
                t-foreach="record.line_ids.mapped('matched_credit_ids.credit_move_id.move_id').filtered(lambda move: move.move_type in ['out_refund'])"
```
This will restrict us to credit notes.